.github/workflows/test-suite.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: bootstrap
       run: ./bootstrap
     - name: configure
-      run: ./configure --enable-stack-check
+      run: ./configure --enable-stack-check --enable-address-sanitizer
```

Comment:
Enabling the address sanitizer seems to disable e.g. optimization.  I want to make sure that the CI tests include the configuration that is also shipped to customers.  I think also doing a testsuite run with the sanitizers active makes complete sense, but please only as addition, not as replacement for the existing one.